srml/bridge/src/lib.rs

```diff
@@ -242,7 +246,7 @@ mod tests {
 				MockBridge::tracked_bridges(1),
 				Some(BridgeInfo {
 					last_finalized_block_number: 42,
-					last_finalized_block_hash: test_header.hash(), // FIXME: This is broken
```

Comment:
I don't know the details but is this now _unbroken_?

Reply:
Yes, this is now fixed in this PR. Was a compilation issue.

Reply:
`testing::Header` didn't have a `hash()` method available, which is why it was broken. But yeah, it's fixed now

Reply:
My comment was due to the fact that only looking at the _local scope of the code_, I only identified that `test_header.hash();` now has a variable and no logic has changed and I was like _hmm.. how does that solve the problem?_ 😬